src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerPool.java

```diff
@@ -304,4 +325,57 @@ private void reinstateKeyworker(KeyworkerDto keyworker, List<OffenderKeyworker>
         log.debug("Key worker with staffId [{}] reinstated to pool. New pool size is [{}] and priority Key worker has staffId [{}] and [{}] allocations.",
                 staffId, keyworkerPool.size(), keyworkerPool.first().getStaffId(), keyworkerPool.first().getNumberAllocated());
     }
+
+    // Gets allocations for Key worker that have expired recently (between start of look back period and current time)
+    private int getRecentDeallocations(Long staffId, String prisonId, Duration lookBackDuration) {
```

Comment:
minor - maybe rename to getrecentDeallocationCount  to make method calls more readable